logstash-core/src/main/java/org/logstash/health/PipelineIndicator.java

```diff
@@ -171,7 +225,7 @@ public Analysis analyze(final Details details) {
                 case UNKNOWN:
                 default:
                     return Analysis.builder()
-                            .withStatus(YELLOW)
+                            .withStatus(UNKNOWN)
```

Comment:
Is it expected change? - `pipeline is not known;` with yellow status sounds strange to me.

Reply:
I'm not sure how the `unknown` pipeline being `YELLOW` slipped through, but I think it was likely part of how we were making sure an unknown pipeline propagate upwards to get past the greens, but now that the order is `GREEN`->`UNKNOWN`->`YELLOW`->`RED` to match Elasticsearch's, the `UNKNOWN` will propagate correctly and we don't need to make it `YELLOW`.